sites/public/src/components/listings/ListingsCombined.tsx

```diff
@@ -16,21 +17,30 @@ const parentStyle = {
   // This is a not-ideal way to do "fill window minus header+footer" however I can't find another way to do this.
   // TODO: update header+footer to a not-magic number
   height: "calc(100vh - 275px)",
+  position: "relative",
 }
 
-const ListingsCombined = (props: ListingsCombinedProps) => (
-  <div className="listings-combined" style={parentStyle}>
-    <div style={{ flex: "1" }}>
-      <ListingsMap listings={props.listings} googleMapsApiKey={props.googleMapsApiKey} />
+const ListingsCombined = (props: ListingsCombinedProps) => {
+  const mapRef = React.useRef(null)
+  return (
+    <div className="listings-combined" style={parentStyle}>
+      <div style={{ flex: "1", position: "relative" }}>
+        <ListingsMap
+          listings={props.listings}
+          googleMapsApiKey={props.googleMapsApiKey}
+          mapRef={mapRef}
+        />
+        <MapControl mapRef={mapRef} />
```

Comment:
Can MapControl live within ListingsMap instead of at this level?

Reply:
Is there an advantage to attempting to get that to work?

Reply:
I believe so. <ListingsMap> is now an unusable component on its own becuase it does not have the ability to zoom in or out, and there is no reason why the parent component needs to know about the state of the MapControl